.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/dmlc/dmlc-core
 [submodule "rabit"]
 	path = rabit
-	url = https://github.com/dmlc/rabit
+	url = https://github.com/chenqin/rabit
```

Comment:
can we revert this?

Reply:
@CodingCat We need to merge https://github.com/dmlc/rabit/pull/106 first.

Reply:
will swing back after https://github.com/dmlc/rabit/pull/106 and https://github.com/chenqin/rabit/commit/edac14220669bab9c3605ee1664d3bce816dbf3c merged

Reply:
and this change https://github.com/dmlc/rabit/pull/120